internal/server/statvar/statvar_summary.go

```diff
@@ -16,22 +16,42 @@ package statvar
 
 import (
 	"context"
+	"database/sql"
 
 	pb "github.com/datacommonsorg/mixer/internal/proto"
+	"github.com/datacommonsorg/mixer/internal/sqldb/sqlquery"
 	"github.com/datacommonsorg/mixer/internal/store"
 	"github.com/datacommonsorg/mixer/internal/store/bigtable"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	"google.golang.org/protobuf/proto"
 )
 
+// GetStatVarSummaryHelper is a wrapper to get stat var summary.
 // GetStatVarSummaryHelper is a wrapper to get stat var summary.
 func GetStatVarSummaryHelper(
 	ctx context.Context, entities []string, store *store.Store) (
 	map[string]*pb.StatVarSummary, error) {
+	if store.BtGroup != nil {
+		return btGetStatVarSummary(ctx, entities, store.BtGroup)
+	}
+	if store.SQLClient != nil {
+		return sqlGetStatVarSummary(entities, store.SQLClient)
+	}
```

Comment:
Can you also merge here, so we don't make the assumption that mixer only uses one storage type? 

Then you can add a golden test for this method by adding data in test/datacommons.db and update the golden response (can check my previous PR on adding golden test).

Reply:
Done.